site/reference/schema/kptfile/kptfile.yaml

```diff
@@ -159,6 +159,28 @@ definitions:
         x-go-name: Namespace
```

Comment:
Is `origin` property not added to generated Kptfile schema?

Reply:
:+1: Needed to run `make schema` --- thank you for noticing that!